lab-kyle/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "tcp chat server",
+  "version": "1.0.0",
+  "description": "![cf](https://i.imgur.com/7v5ASc8.png) lab-05-tcp-chat-server ======",
+  "main": "server.js",
+  "scripts": {
+    "test":
```

Comment:
This is invalid JSON, put in an empty sting or something like "echo \"No tests specified.\""